packages/slate-editor/src/modules/editor-v4-story-embed/components/FloatingStoryEmbedInput/FloatingStoryEmbedInput.scss

```diff
@@ -0,0 +1,30 @@
+@use "sass:math";
```

Comment:
we now use css modules, take a look on packages/slate-editor/src/components/Button for usage example